tools/build-cocoa.sh

```diff
@@ -96,3 +96,59 @@ else
     rm -f "realm-core-cocoa-${v}.tar.xz"
     tar -cJvf "realm-core-cocoa-${v}.tar.xz" core
 fi
+
+# produce xcframeworks
+rm -rf xcframework
+mkdir temp
+make_core_xcframework="xcodebuild -create-xcframework"
+make_core_debug_xcframework="xcodebuild -create-xcframework"
+for bt in "${BUILD_TYPES[@]}"; do
+    [[ "$bt" = "Release" ]] && suffix="" || suffix="-dbg"
+    for p in "${PLATFORMS[@]}"; do
+        if [[ "$p" == "macosx" || "$p" == "maccatalyst" ]]; then
+            cp build-${p}-${bt}/src/realm/librealm${suffix}.a temp/librealm-${p}${suffix}.a
```

Comment:
Why the copy here? Can't the original path just be passed to xcodebuild?

Reply:
I agree its overkill, removed.